### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by doczoidberg** [aaronpowell/glimpse-knockout#2]
Is glimpse-knockout maintained in future?

It crashes when i want to apply knockout bindings from a breeze.js viewmodel.

**Comment by aaronpowell** [comment]
Yep I’ve got some bug fixes for a new release but I too have hit the problem with Breeze.js that I’m trying to solve. Unfortunately it’s taking a lot longer than I’d expected which has held off the new release.
The problem with Breeze.js is that when I try and serialize it for Glimpse to display the object model is very interconnected, it’s possible to go from any Breeze.js model to any other within the data context. This causes crazy circular references in Glimpse that ultimately crashes it.

The only solution I know of right now is to map your breeze models to a separate object for knockout binding but that’s not really ideal.

Aaron Powell
MVP - Internet Explorer (Development) | FunnelWeb Team Memberhttp://funnelweblog.com/

http://apowell.mehttp://apowell.me/ | http://twitter.com/slace | Skype: aaron.l.powell | Githubhttp://github.com/aaronpowell/ | BitBuckethttp://hg.apwll.me/

**Comment by doczoidberg** [comment]
Good to hear that this project is not dead. Unfortunately i have no time to digg deeper into the code. 

Maybe a hack helps for the first time so that it does not crash and does not show all data correct.

**Comment by art-ist** [comment]
Any news on that? I seem to have the same problem.

**Comment by aaronpowell** [comment]
TBH it's been at least 2 years since I used knockout on any projects so I probably won't be doing anything on this plugin.
